modules/service/src/main/scala/lucuma/odb/graphql/OdbMapping.scala

```diff
@@ -496,16 +502,18 @@ object OdbMapping {
         unsafeLoadSchema("OdbSchema.graphql") |+| enums.schema
 
       // Our combined type mappings
-      override val typeMappings: List[TypeMapping] =
-        List(
-          FilterTypeMetaMapping,
-          ObsAttachmentTypeMetaMapping,
-          ObsAttachmentFileExtMapping,
-          PartnerMetaMapping,
-          ProposalAttachmentTypeMetaMapping,
-          ProposalStatusMetaMapping,
-          QueryMapping,
-        ) ++ LeafMappings
+      override val typeMappings: TypeMappings =
+        TypeMappings.unsafe(
```

Comment:
This is an incomplete set of mappings that is only capable of serving enum metadata, so we need to mark it as `unsafe` to bypass the validator.